src/main/java/entities/ComputerPlayerFactory.java

```diff
@@ -0,0 +1,14 @@
+package entities;
+
```

Comment:
Perhaps you could add some Javadocs to this class just briefly describing what it does.

Reply:
Sounds good, I'll add Javadocs to all the classes.